### PR TITLE
register app with settings for apns auth key if useAPNsAuthKey is true

### PR DESCRIPTION
**Pull request by imoans**
- add properties for apns auth key to push-credentials.json
- fix registerApp method to use settings for apns auth key if useAPNsAuthKey is true

### CODE REVIEW

File: default-values/non-model-configs/push-credentials.json [hunk 1]
```diff
@@ -1,5 +1,10 @@
 {
   "gcmServerApiKey" : "dummy",
   "apnsCertData"    : "dummy",
-  "apnsKeyData"     : "dummy"
+  "apnsKeyData"     : "dummy",
+  "apnsTokenKeyPath"     : "dummy",
+  "apnsTokenKeyId"     : "dummy",
+  "apnsTokenTeamId"     : "dummy",
+  "apnsBundleId"     : "dummy",
+  "useAPNsAuthKey"     : "dummy"
```
Comment (imoans) [inline default-values/non-model-configs/push-credentials.json]: fix indent

File: src/server/boot/notification.coffee [hunk 2]
```diff
@@ -12,15 +13,48 @@ module.exports = (app, cb) ->
...
+
+    commonSettings =
+        production: buildInfo.env is 'production'
+        feeedbackOptions:
+            batchFeedback: true
+            interval: 300
+
+    keySettings = {}
+
+    if config.useAPNsAuthKey is true
```
Comment (imoans) [inline src/server/boot/notification.coffee]: `if config.useAPNsAuthKey is true` -> `if config.useAPNsAuthKey`

File: default-values/non-model-configs/push-credentials.json (hunk 1 above)
Comment (imoans) [inline default-values/non-model-configs/push-credentials.json]: fix value of useAPNsAuthKey

**Comment by shinout** [comment]
LGTM

**Pull request merged**